bundles/org.openhab.binding.opengarage/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="opengarage"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sample Thing Type -->
+	<thing-type id="opengarage">
+		<label>OpenGarage</label>
+		<description>OpenGarage Binding</description>
+
+		<channels>
+			<channel id="distance" typeId="opengarage-distance" />
+			<channel id="status" typeId="opengarage-status" />
+			<channel id="vehicle" typeId="opengarage-vehicle" />
+		</channels>
+
+		<config-description>
+			<parameter name="host" type="text" required="true">
```

Comment:
```suggestion
			<parameter name="hostname" type="text" required="true">
```

Reply:
I believe this is addressed in 4714fc9b7d